companies/1and1-gb.json

```diff
@@ -1,16 +1,16 @@
 {
     "slug": "1and1-gb",
     "relevant-countries": [
-        "gb"
+        "gs"
     ],
     "categories": [
         "telecommunication",
         "commerce"
     ],
     "name": "1&1 Internet Ltd.",
-    "address": "Discovery House\n154 Southgate Street\nGloucester\nGL1 2EX\nUnited Kingdom",
+    "address": "1&1 Internet Ltd.\nDiscovery House\n154 Southgate Street\nGloucester\nGL1 2EX\nUnited Kingdom",
```

Comment:
🚫 **[recordlint]** <[name-in-address](https://github.com/datenanfragen/data#addresses)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Record includes `name` in the `address`.
```suggestion
    "address": "Discovery House\n154 Southgate Street\nGloucester\nGL1 2EX\nUnited Kingdom",
```